src/header/css/header-account.pcss

```diff
@@ -270,7 +270,6 @@
 	font-weight: 500;
 	margin-block-end: 10px;
 	padding-inline: 10px;
-	word-break: break-all;
```

Comment:
does this break the layout for a long name? I think that's why it was added...

Reply:
Yeah if the first name or the last name by itself is long enough to exceed the width without any spaces or dashes then it would overflow and overlap beyond the container.  If you think it's better to keep this in we can just close this PR.